docs/apworld specification.md

```diff
@@ -1,26 +1,36 @@
-# apworld Specification
+# APWorld Specification
 
 Archipelago depends on worlds to provide game-specific details like items, locations and output generation.
-Those are located in the `worlds/` folder (source) or `<install dir>/lib/worlds/` (when installed).
+These are called "APWorlds".
+They are located in the `worlds/` folder (source) or `<install dir>/lib/worlds/` (when installed).
 See [world api.md](world%20api.md) for details.
+APWorlds can either be a folder, or they can be packaged as an .apworld file.
 
-apworld provides a way to package and ship a world that is not part of the main distribution by placing a `*.apworld`
-file into the worlds folder.
+## .apworld File Format
 
-**Warning:** apworlds have to be all lower case, otherwise they raise a bogus Exception when trying to import in frozen python 3.10+!
+The `.apworld` file format provides a way to package and ship an APWorld that is not part of the main distribution
+by placing a `*.apworld` file into the worlds folder.
 
-
-## File Format
-
-apworld files are zip archives, all lower case, with the file ending `.apworld`.
+`.apworld` files are zip archives, all lower case, with the file ending `.apworld`.
 The zip has to contain a folder with the same name as the zip, case-sensitive, that contains what would normally be in
 the world's folder in `worlds/`. I.e. `worlds/ror2.apworld` containing `ror2/__init__.py`.
 
+**Warning:** `.apworld` files have to be all lower case,
+otherwise they raise a bogus Exception when trying to import in frozen python 3.10+!
 
 ## Metadata
 
-Metadata about the apworld is defined in an `archipelago.json` file inside the zip archive.
-The current format version has at minimum:
+Metadata about the APWorld is defined in an `archipelago.json` file.
+
+If the APWorld is a folder, the only required field is "game":
+```json
+{
+  "game": "Game Name"
+}
+```
+
+If the APWorld is packaged as an .apworld zip file, it also needs to have `version` and `compatible_version`,
+which refer to the version of the APContainer packaging scheme defined in [Files.py](../worlds/Files.py):
```

Comment:
Is it worth mentioning the build tool we're supposed to use? so no custom world devs get the idea that we have to add these manually during our release processes?